src/main/groovy/com/readytalk/gradle/plugins/ci/CiPublishingPlugin.groovy

```diff
@@ -20,8 +20,9 @@ class CiPublishingPlugin implements Plugin<Project>, PluginUtils {
 
   void apply(final Project project) {
     this.project = project
+    project.plugins.apply(BasePlugin)
 
-    withAnyPlugin(['ivy-publish', 'maven-publish']) {
+    withAnyPlugin(['ivy-publish', 'maven-publish', 'com.gradle.plugin-publish']) {
```

Comment:
That is good to know. I think I will refactor this a bit then. I guess I sort of question the usefulness of the `withAnyPlugin` helper if that is the behavior.